### PR TITLE
Remove Rails.application.secrets in favor of ENV-backed config.

### PR DESCRIPTION
**Pull request by plural**
This resolves the remaining Rails 7.2 deprecation warnings and should simplify things a bit for us since we can use the .env on the deployed environments to control the app.

### CODE REVIEW

File: app/assets/javascripts/nrdb_cards.js.coffee.erb [hunk 1]
```diff
@@ -47,6 +47,7 @@ $(document).on 'turbolinks:load', ->
 
     getPrintings = (request) =>
       request.data['fields[printings]'] = 'card_id,card_type_id,title,side_id,faction_id,minimum_deck_size,influence_limit,influence_cost'
+      1+1
```
Comment (haradan) [inline app/assets/javascripts/nrdb_cards.js.coffee.erb:50]: Was this committed accidentally?

File: config/environments/production.rb [hunk 2]
```diff
@@ -84,8 +84,14 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.abr_host = 'https://alwaysberunning.net'
-  config.nrdb_api_host = 'https://api.netrunnerdb.com'
+  config.abr_host = ENV['ABR_HOST']
```
Comment (haradan) [inline config/environments/production.rb:87]: It looks like this environment variable didn't exist before. Should we default the ABR host to what it was before?

File: config/environments/production.rb [hunk 3]
```diff
@@ -84,8 +84,14 @@
...
+  config.abr_auth = ENV['ABRAUTH']
+  config.nrdb = {
+    client_id: ENV['NRDB_CLIENT'],
+    client_secret: ENV['NRDB_SECRET'],
+    redirect_uri: ENV['NRDB_REDIRECT_URI']
+  }
+  config.nrdb_api_host = ENV['NRDB_API_HOST']
```
Comment (haradan) [inline config/environments/production.rb:94]: It looks like this environment variable didn't exist before. Should we default the NRDB API host to what it was before?

File: app/assets/javascripts/nrdb_cards.js.coffee.erb (hunk 1 above)
Comment (plural) [reply app/assets/javascripts/nrdb_cards.js.coffee.erb:50]: yep! i put that in to force an asset rebuild.  Reverted.

**Review by haradan — APPROVED** [review]

**Pull request merged**